cmd/antrea-controller/options_test.go

```diff
@@ -0,0 +1,178 @@
+				ClusterCIDRs:         []string{"10.10.0.0", "a:b::0/64"},
+				ServiceCIDR:          "172.16.0.0/16",
+				ServiceCIDRv6:        "2620:124::0/64",
+				NodeCIDRMaskSizeIPv4: 24,
+				NodeCIDRMaskSizeIPv6: 70,
+			},
+			expectedErr: "cluster CIDRs [10.10.0.0 a:b::0/64] is invalid",
+		},
+		{
+			name: "invalid ClusterCIDRs num",
+			nodeIPAMConfig: controllerconfig.NodeIPAMConfig{
+				EnableNodeIPAM:       true,
+				ClusterCIDRs:         []string{},
+				ServiceCIDR:          "172.16.0.0/16",
+				ServiceCIDRv6:        "2620:124::0/64",
+				NodeCIDRMaskSizeIPv4: 24,
+				NodeCIDRMaskSizeIPv6: 780,
+			},
+			expectedErr: "at least one cluster CIDR must be specified",
+		},
+		{
+			name: "invalid ClusterCIDRs num",
+			nodeIPAMConfig: controllerconfig.NodeIPAMConfig{
+				EnableNodeIPAM:       true,
+				ClusterCIDRs:         []string{"10.10.0.0/16", "a:b::0/64", "20.20.0.0/24"},
+				ServiceCIDR:          "172.16.0.0/16",
+				ServiceCIDRv6:        "2620:124::0/64",
+				NodeCIDRMaskSizeIPv4: 24,
+				NodeCIDRMaskSizeIPv6: 80,
+			},
+			expectedErr: "at most two cluster CIDRs may be specified",
+		},
+		{
+			name: "at most one cluster CIDR of each type",
+			nodeIPAMConfig: controllerconfig.NodeIPAMConfig{
+				EnableNodeIPAM:       true,
+				ClusterCIDRs:         []string{"10.10.0.0/16", "20.20.0.0/24"},
+				ServiceCIDR:          "172.16.0.0/16",
+				ServiceCIDRv6:        "2620:124::0/64",
+				NodeCIDRMaskSizeIPv4: 24,
+				NodeCIDRMaskSizeIPv6: 80,
+			},
+			expectedErr: "at most one cluster CIDR may be specified for each IP family",
+		},
+		{
+			// 	ipamIPv4MaskLo is 16, ipamIPv4MaskHi is 30
+			name: "invalid node IPv4 CIDR mask size",
+			nodeIPAMConfig: controllerconfig.NodeIPAMConfig{
+				EnableNodeIPAM:       true,
+				ClusterCIDRs:         []string{"10.10.0.0/16", "a:b::0/64"},
+				ServiceCIDR:          "172.16.0.0/16",
+				ServiceCIDRv6:        "2620:124::0/64",
+				NodeCIDRMaskSizeIPv4: 15,
+				NodeCIDRMaskSizeIPv6: 80,
+			},
+			expectedErr: "node IPv4 CIDR mask size 15 is invalid, should be between 16 and 30",
+		},
+		{
+			//	ipamIPv6MaskLo is 64, ipamIPv6MaskHi is 126
+			name: "invalid node IPv6 CIDR mask size",
+			nodeIPAMConfig: controllerconfig.NodeIPAMConfig{
+				EnableNodeIPAM:       true,
+				ClusterCIDRs:         []string{"10.10.0.0/16", "a:b::0/64"},
+				ServiceCIDR:          "172.16.0.0/16",
+				ServiceCIDRv6:        "2620:124::0/64",
+				NodeCIDRMaskSizeIPv4: 24,
+				NodeCIDRMaskSizeIPv6: 63,
+			},
+			expectedErr: "node IPv6 CIDR mask size 63 is invalid, should be between 64 and 126",
+		},
+		{
+			name: "invalid service CIDR",
+			nodeIPAMConfig: controllerconfig.NodeIPAMConfig{
+				EnableNodeIPAM:       true,
+				ClusterCIDRs:         []string{"10.10.0.0/16", "a:b::0/64"},
+				ServiceCIDR:          "1",
+				ServiceCIDRv6:        "2620:124::0/64",
+				NodeCIDRMaskSizeIPv4: 24,
+				NodeCIDRMaskSizeIPv6: 80,
+			},
+			expectedErr: "service CIDR 1 is invalid",
```

Comment:
Could you check and update the lower case `service` and `node` to `Service` and `Node` in fmt.Errorf()?
I think it's allowed to start with upper case when the first letter is nun in the error message. @tnqn @jianjuns any opinions?